mantis-runtime/src/main/java/io/mantisrx/runtime/core/MantisStreamImpl.java

```diff
@@ -129,8 +129,8 @@ private MantisJobBuilder makeMantisJob(ImmutableValueGraph<OperandNode<?>, Manti
         MantisJobBuilder jobBuilder = new MantisJobBuilder();
         final AtomicReference<FunctionCombinator<?, ?>> composite = new AtomicReference<>(new FunctionCombinator<>(false));
         for (OperandNode<?> n : operandNodes) {
-            Set<OperandNode<?>> nbrs = graphDag.successors(n);
-            if (nbrs.size() == 0) {
+            Set<OperandNode<?>> successorsNodes = graphDag.successors(n);
+            if (successorsNodes.size() == 0) {
                 continue;
```

Comment:
what's the need for changing this variable name?

Reply:
@liuml07 suggested preferring readability over more concise code in the issue.